website/ibm.erb

```diff
@@ -195,9 +195,12 @@
             <li<%= sidebar_current("docs-ibm-datasource-dns-resource-records") %>>
               <a href="/docs/providers/ibm/d/private_dns_resource_records.html">dns_resource_records</a>
             </li>
+            <li<%= sidebar_current("docs-ibm-datasource-dns-glb-monitors") %>>
+              <a href="/docs/providers/ibm/d/private_dns_glb_monitors.html">dns_glb_monitors</a>
+            </li>
           </ul>
         </li>
-        <li<%= sidebar_current("docs-ibm-datasource-function") %>>
+        <li<%= sidebar_current("docs-ibm-datasource-function") %>>n
```

Comment:
remove `n` from here.  do not edit exiting lines if it is unnecessary

Reply:
Done